src/GUI/RevEng.Core/README.md

```diff
@@ -0,0 +1,45 @@
+You need to do some setup for your app after the `Reverse Engineer` created the Entity Model classes for you. Here are some hints:
+
+## ASP.NET Core 6.0
+
+1. Register your data context class to `IServiceCollection` in your `Program.cs` file.
+
+    ```cs
+    builder.Services.AddDbContext<[ContextName]>(options => {

```

Comment:
Maybe use AddSqlServer instead?

Reply:
Sorry, it is UseSqlServer.

Reply:
builder.Services.AddSqlServer (new in .NET 6)

Reply:
I don't know that! I know now. But I doubted any other provider also have this? For better compatibility, maybe we can still use the old one.

Reply:
All current providers have this.

Reply:
Nice! I updated the doc.